corehq/apps/reports/filters/users.py

```diff
@@ -410,3 +429,66 @@ def get_user_toggle(request):
     elif group or individual:
         show_filter = False
     return toggle, show_filter
+
+
+class SubmitHistoryFilter(ExpandedMobileWorkerFilter):
+    options_url = 'emwf_options_all_users'
+
+    @property
+    @memoized
+    def utils(self):
+        return SubmitHistoryUtils(self.domain)
+
+    @classmethod
+    def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
```

Comment:
My previous comment on this got buried:

> This is a very large method to duplicate so much - it would be a pain to update both versions. I think you can cheat a little here and just make the modifications to the base class. If "DEACTIVATED" isn't an option, then it'll always be False anyways, and the behavior will be the same.

That's my only remaining piece of feedback, I believe.

Reply:
Ah oops, thanks for catching that:
https://github.com/dimagi/commcare-hq/pull/21564/commits/ecb419ccfcc08cbee6a3e5a42fcd36259c4a53e8

Reply:
Awesome, thanks!